setup/db/db/schema-4920to41000.sql

```diff
@@ -232,3 +232,5 @@ WHERE (o.cpu is null AND o.speed IS NULL AND o.ram_size IS NULL) AND
 
 -- CLOUDSTACK-9827: Storage tags stored in multiple places
 DROP VIEW IF EXISTS `cloud`.`storage_tag_view`;
+
+DELETE FROM `cloud`.`configuration` where name = 'concurrent.snapshots.threshold.perhost';
```

Comment:
@kishankavala Updated. Not deleting this param as it might impact the existing config.